samples/librats-client/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(librats-client)
+
+# /usr/local
+set(RATS_INSTALL_PATH "/usr/local")
+# include/rats
+set(RATS_INSTALL_INCLUDE_PATH "${RATS_INSTALL_PATH}/include/")
+
```

Comment:
There is no need to set RATS_INSTALL_PATH since cmake/CustomInstallDirs.cmake has already set it